### ISSUE TITLE
Summary of validation errors when there are none

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1757]
Originally reported on Google Code with ID 1757

```
There is no need in logging string "Summary of validation erros" if none
are occured.

16:50:01: Copying the database file
16:50:02: Test database
Summary of validation errors 
16:50:03: Checking database NULL values
16:50:04: Commenting stored procedures
16:50:05: Commenting triggers
16:50:07: Archiving views and computed fields
16:50:07: Backup database
16:50:09: Restore database
16:50:14: Restores and computed fields
16:50:15: Removing comments from stored procedures
16:50:17: Removing comments from triggers

```

Reported by `gs1994` on 2009-12-14 14:57:14

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2009-12-22 15:59:31

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2010-01-17 16:33:29
- **Labels added**: FDBConvert
- **Labels removed**: FBDBConvert

**Comment by gsbelarus** [comment]
Reported by `flakekun` on 2010-01-26 13:43:27
- **Status changed**: `Duplicate`
- **Merged into**: #1814

**Comment by gsbelarus** [comment]
Reported by `gs1994` on 2015-06-29 14:08:54
